04_snake_game/main.go

```diff
@@ -69,7 +87,18 @@ func drawSnakePosition(g game) {
 	str := fmt.Sprintf("(%d, %d)", g.sn.pos.x, g.sn.pos.y)
 	writeText(g.fieldWidth-len(str), 0, str, snakeFgColor, snakeBgColor)
 }
+func drawApllePosition(g game) {
+	str := fmt.Sprintf("(%d, %d)", g.ap.pos.x, g.ap.pos.y)
+	writeText(g.fieldWidth-len(str), 0, str, appleFgColor, appleBgColor)
+}
 
+// drawSnake draws the snake in the buffer.
+func drawSnake(sn snake) {
+	termbox.SetCell(sn.pos.x, sn.pos.y, snakeBody, snakeFgColor, snakeBgColor)
+}
```

Comment:
`drawSnake` is declared twice. This is a compilation error. Can you please fix it?